src/WCSimPhysicsList.cc

```diff
@@ -66,7 +66,7 @@ void WCSimPhysicsList::ConstructProcess()
 {
   //AddTransportation();
   ConstructEM();
-  ConstructlArStepLimiter();
+  // ConstructlArStepLimiter();
```

Comment:
Why is this line commented out? Is it necessary for it to be removed for Geant4.10?